src/jobs/deploy_env.yml

```diff
@@ -106,9 +106,26 @@ steps:
   - when:
       condition: <<parameters.slack_notification>>
       steps:
+        - run:
+            name: Slack channels to notify
+            command: |
+              # For prod releases only always notify central dps-releases channel in addition to custom team channels.
+              if [[ "<< parameters.env >>" == "prod" ]] || [[ "<< parameters.env >>" == "production" ]]; then
+                # By default send to channel ID CVA3MKDTR = #dps-releases
+                if [[ "<< parameters.slack_channel_name >>" == "CVA3MKDTR" ]] || [[ "<< parameters.slack_channel_name >>" == "dps-releases" ]]; then
+                  export NOTIFY_SLACK_CHANNELS="CVA3MKDTR"
```

Comment:
don't think you need to `export` here since you're using the variable on line 125 anyway